js/action-kit-service.js

```diff
@@ -0,0 +1,48 @@
+function fetchActionKitData(campaignsString, callback) {
+  if (!campaignsString) {
+    callback({});
+    return;
+  }
+
+  var separator = '|';
+  var campaigns = campaignsString.split(separator);
+  var template =
+    '<div class="popup_box"> ' +
+    '<div class="popup_box_header"> ' +
+    '<strong><a href="{{ rsvp_url }}">{{ name }}</a></strong> ' +
+    '</div> <em>{{ city }}{{#if state }}, {{/if }}{{ state }}{{#if country }},{{/if }} {{ country }}</em> <hr /> ' +
+    '{{ start_time }} <br> {{ event_date }} <br><br> ' +
+    '{{ venue }} <br> {{ address }} <br> {{ city }}{{#if state }}, {{/if }}{{ state }}{{#if country }},{{/if }} {{ country }} <br><br>' +
+    '<strong><a href="{{ rsvp_url }}">RSVP</a></strong> </div>';
+  var compiledTemplate = Handlebars.compile(template);
+  var layerGroups = {};
+  var actionKitRequests = [];
+
+  campaigns.forEach(function(campaign) {
+    actionKitRequests.push($.ajax({
+      url: 'https://act.350.org/event/' + campaign + '?template_set=json_nearby_events&jsonp=?',
```

Comment:
Is there any way to avoid hardcoding the "act.350.org" domain?  Are there conventions or best practices for doing some kind of "package configuration" in a config.js file or something?  Just seems like it would be nice to get this farther out from the code, for other actionkit users' benefit, as long as we've opensourced this.